draft-ietf-httpbis-priority.md

```diff
@@ -755,6 +755,23 @@ practical barriers to determining reasonable stream concurrency limits depending
 on the information that is available to them from the QUIC transport layer.
 TODO: so what can we suggest?
 
+# Retransmission Considerations in QUIC Transport
+
+{{!I-D.ietf-quic-transport}} states endpoints SHOULD prioritize retransmission
+over sending new data, unless otherwise specified by application. When priority
+scheme presented in this document is used, QUIC transport layer MAY take stream
+priorities into account when it has both retransmission data and new data to
+send. For example, if one stream with higher urgency has new data to send and 
```

Comment:
```suggestion
over sending new data, unless otherwise specified by application. When an HTTP/3
application uses the priority scheme defined in this document, the QUIC transport layer
MAY take stream priorities as input to scheduling the sending of both retransmission
data and new data. For example, if one stream with higher urgency has new data to send and
```